docs/docs/cmd/aad/o365group/o365group-add.md

```diff
@@ -20,10 +20,10 @@ m365 aad o365group add [options]
 : Name to use in the group e-mail (part before the `@`)
 
 `--owners [owners]`
-: Comma-separated list of Microsoft 365 Group owners
+: Comma-separated list of valid Microsoft 365 Group owners
```

Comment:
I don't think we need to mention this specifically, as all input we expect should be valid.